pkg/globalnet/controllers/ipam/gatewaymonitor.go

```diff
@@ -326,3 +337,41 @@ func (gm *GatewayMonitor) stopIpamController() {
 		cleanup.ClearGlobalnetChains(gm.ipt)
 	}
 }
+
+func ConfigureTcpMTUProbeValue(mtuProbe, mssValue []byte) {
+	// If we are unable to update the values, just log a warning, Globalnet functionality works fine except
+	// for one use-case where Pod with HostNetworking on Gateway node has mtu issues connecting to remoteServices.
+	if err := ioutil.WriteFile(tcpMtuProbingProcEntry, mtuProbe, 0644); err == nil {
```

Comment:
For the unit tests, I think it would be better to abstract this functionality behind an interface.  I propose to make `MTUConfig` an interface with the `ReadTcpMTUConfigurationOnNode` and `ConfigureTcpMTUProbeValue` methods and make it configurable on `GatewayMonitor`.  If this sounds reasonable, I could do it in a follow-up if you want.